### DISCUSSION TITLE
Cloning Repository Problem

### DISCUSSION THREAD

**Discussion by 86-ark** [horilla-opensource/horilla#266] [Q&A]
![Horilla hr](https://github.com/user-attachments/assets/a15c311b-189e-4e27-ba2a-39ceb6f307bd)

**Comment by horilla-opensource** [comment]
> ![Horilla hr](https://private-user-images.githubusercontent.com/84174059/352812841-a15c311b-189e-4e27-ba2a-39ceb6f307bd.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyMjk0NDMsIm5iZiI6MTcyMjIyOTE0MywicGF0aCI6Ii84NDE3NDA1OS8zNTI4MTI4NDEtYTE1YzMxMWItMTg5ZS00ZTI3LWJhMmEtMzljZWI2ZjMwN2JkLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDA0NTkwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZkZThkMjlhZjczZWViM2NmNzEwNTY4MTdiMjYwZTBkYzkwZjQ5MDU1OTU1MmNjZjUyOTg2OTBlOTJjNjM0ZmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aD6xLgoDng4KHFOKfBSQxdNSLk1mPzcAT6GJcmuapDM)

Hi @86-ark ,
This is an issue with the git and not the repository.
Please run the below command and try again.
```
git config --global core.compression 9
```

**Reply by horilla-opensource** [reply]
Also please refer to: 
[fatal: early EOF fatal: index-pack failed](https://stackoverflow.com/questions/21277806/fatal-early-eof-fatal-index-pack-failed)

**Comment by 86-ark** [comment]
Thanks
